client/state/state_database.go

```diff
@@ -36,6 +40,14 @@ var (
 	// allocations -> $allocid -> $taskname -> the keys below
 	taskLocalStateKey = []byte("local_state")
 	taskStateKey      = []byte("task_state")
+
+	// devManagerBucket is the bucket name containing all device manager related
+	// data
+	devManagerBucket = []byte("devicemanager")
+
+	// devManagerPluginStateKey is the key serialized device manager
+	// plugin state is stored at
+	devManagerPluginStateKey = []byte("plugin-state")
```

Comment:
underscore to match other bucket names
```suggestion
	devManagerPluginStateKey = []byte("plugin_state")
```